airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -19,12 +19,37 @@
         },
         "uniqueItems": true
       },
-      "access_token": {
-        "title": "Access Token",
-        "description": "The value of the generated access token. From your App\u2019s Dashboard, click on \"Marketing API\" then \"Tools\". Select permissions <b>ads_management, ads_read, read_insights, business_management</b>. Then click on \"Get token\". See the <a href=\"https://docs.airbyte.com/integrations/sources/facebook-marketing\">docs</a> for more information.",
-        "order": 1,
-        "airbyte_secret": true,
-        "type": "string"
+      "credentials": {
+        "title": "authentication credentials",
```

Comment:
Why does this differ from `spec.py`? https://github.com/airbytehq/airbyte/pull/38304/files#diff-465a935c5dc0fc41f783e4f82ca3ba4bacf4d0dd97065fa273d7a7bdad94ac9bR33